signed/verifiers.go

```diff
@@ -157,6 +162,9 @@ type RSAPyCryptoVerifier struct{}
 // with PyCrypto.
 func (v RSAPyCryptoVerifier) Verify(key data.PublicKey, sig []byte, msg []byte) error {
 	digest := sha256.Sum256(msg)
+	if key.Algorithm() != data.RSAKey {
+		return ErrInvalid
+	}
```

Comment:
we should verify that len(byte([]key.Public())) is larger than 2048 (our minimum RSA key size)